src/mlpack/methods/decision_tree/all_categorical_split.hpp

```diff
@@ -47,6 +47,7 @@ class AllCategoricalSplit
    * @param numClasses Number of classes in the dataset.
    * @param minimumLeafSize Minimum number of points in a leaf node for
    *      splitting.
+   * @param maximumDepth Maximum Depth Minimum for the tree.
```

Comment:
I found this description somewhat doubtful. Maybe a typo?

Reply:
Sorry i will remove this in the next commit.